lib/matplotlib/colorbar.py

```diff
@@ -294,6 +228,70 @@ def draw(self, renderer):
         return ret
 
 
+class _ColorbarAxesLocator:
+    """
+    Wrap any locator on the axes, or its position, to shrink the
+    inner axes if there are extend triangles at either min or max.
```

Comment:
> Suggest removing the `or its position` because you're only initializing this with the cbar.

The more common case is no locator I think...

> I also wouldn't call this `inner axes` anymore. You only have one axes now.

:+1:

Reply:
I think the word "Wrap" is a bit confusing here then. Perhaps something like "A locator used to position the axes of the Colorbar and shrink, if necessary, to account for the extend triangles."

Reply:
Ooops, sorry, some edits got lost when I had a rebase issue.  Let me go through and make sure these are all still addressed...